lib/modules/commentDepth.js

```diff
@@ -39,9 +44,14 @@ module.options = {
 		}, {
 			name: 'commentDepth',
 			type: 'text',
+			value: '4',
+		}, {
+			name: 'minimumComments',
+			type: 'text',
+			value: '50',
```

Comment:
Yeah, it's a smallish module and I don't expect it to scope creep too much.

As an academic exercise, I would implement it like:

```javascript
module.loadDynamicOptions = function() {
    module.options.subredditDepthsWhateverItsCalled.fields[0].value = module.options.defaultDepth.value;
    module.options.subredditDepthsWhateverItsCalled.fields[1].value = module.options.defaultMinimum.value;
};

```